recipe/meta.yaml

```diff
@@ -108,86 +121,64 @@ outputs:
+        - icu
+        - pcre2
+        - tk
+        - zlib
+        - gsl
+        - libtiff
+        - libblas                            # [not win]
+        - liblapack                          # [not win]
         # sed is in host env because its path is recorded in R's shell wrapper
         - sed                                # [linux]
       run:
         - _r-mutex 1.* anacondar_1
         # Needed by R CMD javareconf
         - make                               # [not win]
-        - gcc_impl_{{ target_platform }}  >={{ c_compiler_version|default("0") }}    # [linux]
-        - gxx_impl_{{ target_platform }}  >={{ cxx_compiler_version|default("0") }}  # [linux]
-        - gfortran_impl_{{ target_platform }}  # [linux]
+        - gcc_impl_{{ target_platform }}  >={{ c_compiler_version|default("0") }}    # [linux or win]
+        - gxx_impl_{{ target_platform }}  >={{ cxx_compiler_version|default("0") }}  # [linux or win]
+        - gfortran_impl_{{ target_platform }}  # [linux or win]
         - {{ c_compiler|default("blah") }}_{{ target_platform }} >={{ c_compiler_version|default("0") }}      # [osx]
         - {{ cxx_compiler|default("blah") }}_{{ target_platform }} >={{ cxx_compiler_version|default("0") }}  # [osx]
         - {{ compiler('fortran') }}  # [osx]
-        - curl                                 # [not win]
-        - {{ native }}bzip2                    # [win]
-        - {{ native }}libjpeg-turbo            # [win]
-        - {{ native }}libiconv                 # [win]
+        - {{ pin_compatible('llvm-openmp', max_pin=None) }}      # [osx]
+        - curl
+        - libuuid                              # [linux]
         - xorg-libxt                           # [linux]
-        - {{ native }}gmp                      # [win]
-        - {{ native }}fftw                     # [win]
-        - {{ native }}xz                       # [win]
-        - {{ native }}mpfr                     # [win]
-        - {{ native }}libsndfile               # [win]
-        - {{ native }}bwidget
-        - {{ native }}tktable
-        - {{ native }}libpng                   # [win]
-        - {{ native }}pcre2                    # [win]
-        - {{ native }}tk                       # [win]
-        - {{ native }}zlib                     # [win]
-        - {{ native }}libtiff                  # [win]
-        - sed                                # [linux]
+        - bwidget
+        - tktable
+        - tk
+        - sed                                  # [linux]
 
     test:
       commands:
+        - echo on  # [win]
         - R -h
         - R --version
-        - R -e "print(.Platform\$dynlib.ext)"
-        - R -e "library('tcltk')"            # [linux or osx]
-        - R -e \"library('tcltk')\""         # [win]
+        - R -e 'print(.Platform$dynlib.ext)' | grep -F -- "${SHLIB_EXT}"  # [not win]
+        - R -e "print(.Platform$dynlib.ext)" | findstr "/C:%SHLIB_EXT%"   # [win]
+        - R -e "library('tcltk')"
```

Comment:
This now gives
```
2024-06-22T01:43:27.9374519Z > library('tcltk')
2024-06-22T01:43:28.5003088Z error reading package index file D:/bld/r-base-split_1719018376064/_test_env/Library/lib/Tktable/pkgIndex.tcl: expected version number but got "package require Tk 8.2; load D:/bld/r-base-split_1719018376064/_test_env/Library/lib/Tktable/Tktable.dll Tktable"
2024-06-22T01:43:28.5556642Z error reading package index file D:/bld/r-base-split_1719018376064/_test_env/lib/R/../../Library/lib/Tktable/pkgIndex.tcl: expected version number but got "package require Tk 8.2; load D:/bld/r-base-split_1719018376064/_test_env/lib/R/../../Library/lib/Tktable/Tktable.dll Tktable"
```
on Windows.

Reply:
`tktable=2.10=h7534570_5`:
```tcl
# cat Library/lib/Tktable/pkgIndex.tcl 
if {[catch {package require Tcl 8.2}]} return
package ifneeded Tktable  "package require Tk 8.2; [list load [file join $dir Tktable.dll] Tktable]"
```
`m2w64-tktable=2.10=5` (source of Tktable in `r-base=4.1.3`):
```tcl
# cat Library/mingw-w64/lib/Tktable2.10/pkgIndex.tcl 
if {[catch {package require Tcl 8.2}]} return
package ifneeded Tktable 2.10 \
    [list load [file join $dir Tktable210.dll] Tktable]
```

Reply:
I can't see anything too special in https://github.com/msys2/MINGW-packages/blob/ec15eebeef4f00f9347cb698e84351d280493645/mingw-w64-tktable/PKGBUILD .
So could be upstream issue with VC nmake files? IDK.

Reply:
That one will be fixed with https://github.com/conda-forge/tktable-feedstock/pull/13 .
It's not so nice that this did not result in a proper test failure...